openedx-unsupported/configuration
# fix: remove dummy order history url

Configuration Pull Request
---

Make sure that the following steps are done before merging:



## Description

This PR changes the default value of `MFE_ORDER_HISTORY_URL` to emptry string `""` from a dummy url configured earlier.

MFE headers [renders](https://github.com/openedx/frontend-component-header/blob/faf1b8522a43a4a995f8f3671e48d25f4ee439ab/src/Header.jsx#L73-L76) the `Order History` option in the dropdown, only if `MFE_ORDER_HISTORY_URL` is configured.

Having the dummy url as default value here meant, the `Order History` option is rendered by default in MFEs which redirects to a broken link. This could be confusing for users.

With this change, the `Order History` option will not be rendered by default, unless the `MFE_ORDER_HISTORY_URL` is configured.

## Supporting information

Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.
Be sure to check they are publicly readable, or if not, repeat the information here.

## Testing instructions

1. Register and login to this [sandbox](https://order-history-url.opencraft.hosting/dashboard).
2. Go to the [account page](https://app.order-history-url.opencraft.hosting/account)
3. Click on username in the right hand corner of the MFE header.
4. Verify `Order History` option is missing.

## Deadline

"None" if there's no rush, or provide a specific date or event (and reason) if there is one.

## Other information

Private Ref: [BB-6761](https://tasks.opencraft.com/browse/BB-6761)
